splice_si_api/src/main/java/com/splicemachine/si/impl/data/StripedTxnLifecycleStore.java

```diff
@@ -136,6 +137,30 @@ public void rollbackTransaction(long txnId) throws IOException{
     }
 
     @Override
+    public void rollbackSubtransactions(long txnId, long[] subIds) throws IOException {
+        long beginTS = txnId & SIConstants.TRANSANCTION_ID_MASK;
+
+        Lock lock=lockStriper.get(beginTS).writeLock();
+        acquireLock(lock);
+        try{
+            Txn.State state=baseStore.getState(beginTS);
+            if(state==null){
+                return;
+            }
```

Comment:
I'm pretty sure this will never happen, the only chance is if txnId refers to a readonly transaction that hasn't been registered on the Txn table. But I'd rather change this in a follow up JIRA just in case it breaks tests or other stuff.